scripts/codegen-firecracker.sh

```diff
@@ -23,11 +29,27 @@ echo
 
 # At the moment, we only test codegen for the virtio module
 cd $KANI_DIR/firecracker/src/devices/src/virtio/
+
+# Clean first
+cargo clean
+
 export KANI_LOG=error
 export RUSTC_LOG=error
 export RUST_BACKTRACE=1
-# Use cargo assess since this is now our default way of assessing Kani suitability to verify a crate.
-cargo kani --enable-unstable --only-codegen assess
+
+# Compile rust to iRep
+RUST_FLAGS=(
+    "--kani-compiler"
+    "-Cpanic=abort"
+    "-Zalways-encode-mir"
+    "-Cllvm-args=--goto-c"
+    "-Cllvm-args=--ignore-global-asm"
+    "-Cllvm-args=--reachability=pub_fns"
+    "--sysroot=${KANI_DIR}/target/kani"
+)
+export RUSTFLAGS="${RUST_FLAGS[@]}"
+export RUSTC="$KANI_DIR/target/kani/bin/kani-compiler"
+$WRAPPER cargo build --verbose --lib --target $TARGET
```

Comment:
Do we want to keep `--verbose` in CI?

Reply:
I think so. It's not that verbose and it helps debugging issues.